CONTRIBUTING.md

```diff
@@ -149,10 +165,18 @@ To enforce the [branching strategy](#branching-strategy) pull requests from `mas
 
 ### Merging
 
+Only maintainers can merge pull requests, so PRs from contributors require that at least one maintainer agrees with the changes.
+Ideally, two should give their thumbs up, though.
+Likewise, PRs opened by a maintainer should be reviewed and agreed to by at least one other maintainer.
+Going further, we should avoid merging PRs that one maintainer outright disagrees with and instead work towards a solution that is acceptable to everybody.
+Note all the _should_-s - exceptions can be made if justifiable (e.g. maintainers don't react or there is reason to hurry).
+
 A pull request is accepted by squashing the commits and fast-forwarding master, making each bug fix or feature appear atomically on master.
 This can be achieved with GitHub's [_squash and merge_](https://help.github.com/articles/about-pull-request-merges/#squash-and-merge-your-pull-request-commits) feature.
 
-To make the single commit expressive its message must be detailed and [good]((http://chris.beams.io/posts/git-commit/)) (really, read that post!).
+### Commit Message
+
+To make the single commit expressive, its message must be detailed and [good]((http://chris.beams.io/posts/git-commit/)) (really, read that post!).
```

Comment:
We should emphasize that the commit messages in the PRs branch really don't matter, because they get squashed anyway. What matters is the commit message when merging - and this in in responsibility by a maintainer.

Reply:
While the commit messages don't show up on `master` it is still helpful to have them during development on the branch. Also it's a good habit to get into. I don't want to give the impression that the messages don't matter.